### DISCUSSION TITLE
How to get current X-Y coords

### DISCUSSION THREAD

**Discussion by wigster** [humbertogontijo/homeassistant-roborock#558] [Q&A]
Hi

I want to set up a script to send the vacuum to a particular place in my house to have me empty the dust bin etc (just a charging dock here). 

Is there some way to extract the XY coordinates of the current vacuum position so I can do this? I keep guessing and usually end up giving a coord inside an obstacle.

**Comment by thibaultmol** [comment]
That's what `roborock.get_vacuum_current_position` is for I think
